### ISSUE TITLE
Please read & provide the following

### ISSUE THREAD

**Issue by MyKagome** [facebookresearch/detr#451]
Dear Sir!
Thank you for your great job.
When I used DETR to detect my own data set, I found that AP was always very low, so I tried to reproduce the results of the paper. Because the coco data set was too large, I only randomly selected 8 pictures and corresponding labels as the training set, and the val set was in the same way.I thought the model would converge quickly, but it doesn't seem to.I visualized the result as follows: 
![000000397133](https://user-images.githubusercontent.com/66203616/136920492-bba9baa9-8373-4333-8c28-112e8acae282.png)
![000000174482](https://user-images.githubusercontent.com/66203616/136920290-a2396c96-f730-4a38-82ac-6971b845e747.png)

![000000252219](https://user-images.githubusercontent.com/66203616/136920390-bfc3d581-53a8-4bbb-a51d-b8307117744e.png)
It
![Uploading 000000397133.png…]()
![000000403385](https://user-images.githubusercontent.com/66203616/136920559-811e67c8-d4f4-45a3-b947-6ce16252a949.png)
 looks like all the output is the same (every box position and category), regardless of the input.I had the same problem with custom cubes. At first I thought there might be an error with custom cubes labels, but the same problem happened with coco cubes. I don't know what the problem is.In addition, I also tried to load the official pre-training weight for fine-tuning, but still, the loss decreased slightly.

Does anyone know what the reason is?Thank you so much for your help!
